Formula/openjdk.rb

```diff
@@ -30,90 +31,100 @@ class Openjdk < Formula
   depends_on "autoconf" => :build
   depends_on xcode: :build if Hardware::CPU.arm?
 
+  if Hardware::CPU.arm?
+    patch do
+      url "https://github.com/openjdk/jdk/commit/ca93399af103384e750dabf3abcc6e8392bcf3f4.patch?full_index=1"
+      sha256 "6bea7673e65b57b673174fdfb5c82754942021da79969ef0d462b9606030a939"
+    end
+  end
```

Comment:
@Moisan sorry if I confused you but the commit is already in `jdk-17+24`, so this can be deleted

Reply:
Got it!